docs/source/markdown/podman-generate.1.md

```diff
@@ -13,7 +13,8 @@ The generate command will create structured output (like YAML) based on a contai
 
 | Command | Man Page                                                   | Description                                                                         |
 |---------|------------------------------------------------------------|-------------------------------------------------------------------------------------|
-| kube    | [podman-generate-kube(1)](podman-generate-kube.1.md)       | Generate Kubernetes YAML based on containers, pods or volumes.                               |
+| kube    | [podman-generate-kube(1)](podman-generate-kube.1.md)       | Generate Kubernetes YAML based on containers, pods or volumes.                      |
+| spec    | [podman-generate-spec(1)](podman-generate-spec.1.md)       | Generate Specgen JSON based on containers or pods.                                  |
```

Comment:
I thought we were going to generate YAML?

Reply:
the API takes JSON. I agree YAML is more ideal for readability but has no use case for how we aim for this to be applied.

Reply:
Is it easy to convert to YAML?  If so could we add a flag to convert it?

Reply:
@baude, thoughts on this? I've been letting this sit while you were away. personally I think just JSON makes the most sense for usage, but if we want to eventually support consumption by a cli command, yaml could work too.

Reply:
Any chance we can have this link to the API docs for Specgen?